root/etc/services.d/consumer/run

```diff
@@ -10,4 +12,3 @@ fi
 
 exec \
     s6-setuidgid abc python3 manage.py document_consumer
-
```

Comment:
Please keep this newline, across all the runfiles

Reply:
this newline isnt in the paperless runfile (https://github.com/linuxserver/docker-paperless-ng/blob/master/root/etc/services.d/paperless/run) and it is actually 2 newlines which I don't think is correct?

Reply:
https://i.imgur.com/uHcNPgX.png

Reply:
@Roxedus what is your verdict on this? should I put the double newlines back in?